docs/UserGuide.md

```diff
@@ -215,7 +219,7 @@ Example:
 
 Sorts the review list by the specified criterion.
 
-Format: `ssort CRITERION`
+Format: `rsort CRITERION`
```

Comment:
Fixes the second error spotted in #282. Fix for the first error pending from @clarence-chew.